packages/manager/apps/web/client/app/hosting/database/DATABASE_LIST.html

```diff
@@ -113,7 +113,14 @@
                         <tr>
                             <th scope="col" data-translate="hosting_tab_DATABASES_table_header_name"></th>
                             <th scope="col" data-translate="hosting_tab_DATABASES_table_header_database_name"></th>
-                            <th scope="col" data-translate="hosting_tab_DATABASES_table_header_server_address"></th>
+                            <th scope="col">
+                                <span id="serverAddressHeader" data-translate="hosting_tab_DATABASES_table_header_server_address"></span>
+                                <span class="fa fa-info-circle"
+                                      aria-hidden="true"
```

Comment:
```suggestion
```
Else screen reader won't be able to access this

Reply:
Done